shared/constants/src/idle-helper/general.ts

```diff
@@ -15,6 +15,5 @@ export const USER_ACC_OFF_ACTIONS = {
   skip: 'skip'
 } as const;
 
-export const SUPPORT_SERVER_INVITE_LINK = 'https://discord.gg/magXasew7C';
-export const BOT_INVITE_LINK =
-  'https://discord.com/api/oauth2/authorize?client_id=1133034852611018783&permissions=274878254144&scope=bot%20applications.commands';
+export const SUPPORT_SERVER_INVITE_LINK = 'https://discord.gg/GYcFwdhMgd';
```

Comment:
Consider this is a open source bot, we can move the invite link to env file. 
For whoever come to this project to look for any active support server, we can list down the server and invite links on README.md